python/cudf/cudf/tests/test_joining.py

```diff
@@ -219,7 +219,7 @@ def test_dataframe_join_combine_cats():
     expect.index = expect.index.astype("category")
     got = lhs.join(rhs, how="outer")
 
-    assert_eq(sorted(expect.index), sorted(got.index))
+    assert_eq(sorted(expect.index), sorted(got.to_pandas().index))
```

Comment:
We should just make `assert_eq` handle Index objects, no?

Reply:
We do but we will need to implement `Index.sort_values` incase of this specific test: https://github.com/rapidsai/cudf/issues/5676

Reply:
👍